example/server.js

```diff
@@ -3,21 +3,28 @@
 const express = require('express')
 const falcorExpress = require('falcor-express')
 const Router = require('falcor-router')
-const falcorPostman = require('./index.js')
+const falcorPostman = require('./../falcor-postman.js')
 const app = express()
+const isDevelopment = process.env.NODE_ENV !== 'production'
+const port = process.env.PORT ? process.env.PORT : 3000
```

Comment:
Yes agree, as const/let  are block scoped and don't hoist. I grouped them here while refactoring, will move them closer to usage.